guides/doc-Managing_Hosts/topics/Synchronizing_Templates.adoc

```diff
@@ -334,6 +334,7 @@ To avoid errors after removing the foreman_templates plugin:
 
 . Uninstall the plug-in:
 +
+[options="nowrap" subs="+quotes,attributes"]
 ----
-# yum remove tfm-rubygem-foreman_templates
+# {package-remove-project} tfm-rubygem-foreman_templates
```

Comment:
This only works on EL7. The package name is different on EL8 and Debian. On RPMs you can use the virtual:
```suggestion
# {package-remove-project} foreman-plugin-templates
```

(different plugin, but same story)
```
[root@foreman ~]# yum remove foreman-plugin-monitoring
Loaded plugins: fastestmirror
Resolving Dependencies
--> Running transaction check
---> Package tfm-rubygem-foreman_monitoring.noarch 0:1.0.1-4.fm2_1.el7 will be erased
--> Finished Dependency Resolution

Dependencies Resolved

======================================================================================================================================================================================
 Package                                                 Arch                            Version                                      Repository                                 Size
======================================================================================================================================================================================
Removing:
 tfm-rubygem-foreman_monitoring                          noarch                          1.0.1-4.fm2_1.el7                            @foreman-plugins                          459 k

Transaction Summary
======================================================================================================================================================================================
Remove  1 Package

Installed size: 459 k
```

Reply:
I think that's a good idea.